docs/tech_specs/pa_message_api.md

```diff
@@ -2,7 +2,7 @@
 
 Lists all PA messages that are currently eligible to play.
 
-**URL** : `/api/active_pa_messages`
+**URL** : `/api/pa_messages/active_pa_messages`
```

Comment:
This seems a bit redundant now it has the prefix. Can it be `/api/pa_messages/active`?

Alternatively, just `/api/pa_messages`, unless there's some more general "index" action we should reserve that for. To my recollection, though, "active" PA messages are the only kind we really care about?

Reply:
Hmm, yeah I didn't notice that before but it is definitely redundant. I think your simplified `api/pa_messages` is a good one. I imagine that any feature needing inactive messages will live in Screenplay so wouldn't need this particular API scope to retrieve them.